slidechain/cmd/slidechaind/slidechaind.go

```diff
@@ -39,6 +41,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	c.S.BlockInterval = *blockInterval
```

Comment:
Slight preference for passing this block interval into `slidechain.GetCustodian` / `newCustodian` and then initializing it in the submitter there (rather than making the `BlockInterval` field public)